vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/create/CreateProjectLayout.java

```diff
@@ -47,6 +51,10 @@ protected VerticalLayout initContent() {
 
     FormLayout formLayout = new FormLayout();
     formLayout.addFormItem(titleField, "Project Title");
+    formLayout.addFormItem(projectObjective, "Project Objective");
+    formLayout.setResponsiveSteps(new ResponsiveStep("0", 1));
+    titleField.setSizeFull();
+
```

Comment:
```suggestion
    projectObjective.setWidthFull();
```
Do you want to increase the size of the objective as well?